presto-native-execution/presto_cpp/main/PeriodicMemoryChecker.cpp

```diff
@@ -18,6 +18,10 @@
 #include "velox/common/caching/AsyncDataCache.h"
 #include "velox/common/memory/Memory.h"
 
+#include <sys/stat.h>
```

Comment:
Are these three new includes still needed?

Reply:
```
#include "velox/common/caching/AsyncDataCache.h"
#include "velox/common/memory/Memory.h"
```
was added by https://github.com/prestodb/presto/pull/22229, the file still uses these includes so cannot remove them

Removing 
```
#include <sys/stat.h>
```
since it was only used in the debug code that is to also now removed in this PR